### PR TITLE
copy edits before CE announcement

### PR DESCRIPTION
**Pull request by mikemorran**
Hey @tanfarming, just tidying up the read me

### CODE REVIEW

File: community-edition/readme.md [hunk 1]
```diff
@@ -1,48 +1,57 @@
+# Introduction
+
+Community Edition is a free tool designed to help developers deploy the full Hubs stack on cloud computing software of their choosing. Community Edition simplifies and automates most of the complex deployment process using Kubernetes, a containerized software orchestration system.
+
+Community Edition is designed for developers capable of working with the full Hubs stack and of navigating complex application infrastructure. With Community Edition, developers are responsible for designing, building, hosting, and maintaining their software throughout its lifetime. In order to create a production-ready version of Hubs similar to Hubs Cloud, Developers using Community Edition will need to implement additional features and customizations outside of those listed in this repo. See "Considerations for Production Environment".
+
+# Why Kubernetes
+
+Kubernetes is an industry standard for allowing developers to build, deploy, and scale application efficiently and reliably. Benefits to Kubernetes include:
```
Comment (tanfarming) [inline community-edition/readme.md]: > Kubernetes is an industry standard for allowing developers to build...

i don't think k8s is that, it's an free opensource container orchestrator
it doesn't host code directly so it's technically incorrect to say that it's for developers to build/deploy/scale -- serverless options on public clouds (ie aws-lambda) are the tools designed and marketed for that usage

File: community-edition/readme.md [hunk 2]
```diff
@@ -1,48 +1,57 @@
...
+
+- [Portability, Extensibility, and Open Source](https://kubernetes.io/docs/concepts/overview/)
+- [Availability in many cloud environments](https://kubernetes.io/docs/setup/production-environment/turnkey-solutions/)
+- Many options for single server deployments (ie. minikube, k3s, microk8s, kind)
+
+# Prerequisites
+
+Before deploying the charts in this repo, you will need to choose and configure the following services...
```
Comment (tanfarming) [inline community-edition/readme.md:17]: a few issues in this sentence -- they are not all services, managed k8s is but local k8s is just a software you install, opening up the ports is also not a service

File: community-edition/readme.md [hunk 3]
```diff
@@ -1,48 +1,57 @@
...
+
+- A hosting service with a Kubernetes cluster to receive your Community Edition deployment spec
```
Comment (tanfarming) [inline community-edition/readme.md:19]: again managed k8s is an option not a requirement, you can install k8s on your phone and deploy CE there
also "a hosting service with a kubernetes cluster" sounds confusing to me, if you are referring to things like eks/gke/aks, i think they are officially referred as "kubernetes as a service"

File: community-edition/readme.md [hunk 4]
```diff
@@ -1,48 +1,57 @@
...
+- Kubernetes controls on your device
```
Comment (tanfarming) [inline community-edition/readme.md:20]: i'm not sure what is this? kubectl?

File: community-edition/readme.md [hunk 5]
```diff
@@ -1,48 +1,57 @@
...
+- A DNS service to reach Hubs on a domain
+- Port to expose services to client
+  - TCP: 80, 443, 4443, 5349
+  - UDP: 35000 -> 60000
+- An SMTP service for login emails and accounts (for login emails, ie. [use gmail as smtp](https://support.google.com/a/answer/176600?hl=en))
+
+# Deploy to Kubernetes
+
+To deploy to your K8s cluster on your chosen hosting solution, follow these steps:
```
Comment (tanfarming) [inline community-edition/readme.md:29]: > your K8s cluster on your chosen hosting solution

i don't think we should stress "on your chosen hosting solution", we assume only k8s, we don't care how the k8s maybe hosted, using the k8s on android phone example -- a phone is not quite a "hosting solution", maybe a "hosting platform" or "hosting device"? anyway i think we should just avoid mentioning things beyond k8s

File: community-edition/readme.md [hunk 6]
```diff
@@ -1,48 +1,57 @@
...
+
+- Add your chosen services into `render_hcce.sh` with a text editor
```
Comment (tanfarming) [inline community-edition/readme.md:31]: they are not services, they are just configs, some are "credentials to access a service", like the stmp creds, but many others like namespaces/domain/email/keys are not services at all, and the pgsql ones maybe "creds to access a service", or not, depends on if a external managed pgsql is used

File: community-edition/readme.md [hunk 7]
```diff
@@ -1,48 +1,57 @@
...
+- Run `bash render_hcce.sh && kubectl apply -f hcce.yaml`
```
Comment (tanfarming) [inline community-edition/readme.md:32]: Run seems to be unecessary, for example https://code.visualstudio.com/docs/setup/linux

File: community-edition/readme.md [hunk 8]
```diff
@@ -54,62 +63,54 @@ gcloud auth login
 `sudo apt update && sudo apt install npm && sudo npm install pem-jwk -g`
 ### install k3s
 ```
-### install k3s without traefik
+
+### Step 2: Install k3s without traefik
```
Comment (tanfarming) [inline community-edition/readme.md:67]: yeah i see the structure of the 2 examples there are kind of messed up, i'll get it fixed shortly

File: community-edition/readme.md [hunk 9]
```diff
@@ -1,48 +1,57 @@
+# Introduction
+
+Community Edition is a free tool designed to help developers deploy the full Hubs stack on cloud computing software of their choosing. Community Edition simplifies and automates most of the complex deployment process using Kubernetes, a containerized software orchestration system.
```
[the remark this answers is not in the document]
Comment (mikemorran) [reply community-edition/readme.md:3]: > this is not correct, CE is an arch for hubs designed and configured in k8s, it didn't simplify much of anything. I was actually aiming for the opposite -- i tried to expose everything to provide user the most freedom. simplification reqires wrapper which adds opacity, while it simplifies things for "users" that treat the system as a blackbox, wrappers/opacities makes it a lot more difficult for "tinkers" that want to look inside the box -- CE is made for "tinkers", with an option for "users" to simply just deploy it by following the examples if it's low level sys-admin chores you were referring to, all the simplification and automation were done between k8s and docker.

My understanding is that Community Edition does, by nature of using K8s, simplify and automate a lot of the deployment process **relative to** our current self-hosting process who have to manage each Hubs service on their own. I think that the edit suggested contextualizes what makes CE unique relative to the Hubs codebase that came before.

File: community-edition/readme.md [hunk 10]
```diff
@@ -1,48 +1,57 @@
...
+
+Community Edition is designed for developers capable of working with the full Hubs stack and of navigating complex application infrastructure. With Community Edition, developers are responsible for designing, building, hosting, and maintaining their software throughout its lifetime. In order to create a production-ready version of Hubs similar to Hubs Cloud, Developers using Community Edition will need to implement additional features and customizations outside of those listed in this repo. See "Considerations for Production Environment".
```
[the remark this answers is not in the document]
Comment (mikemorran) [reply community-edition/readme.md:5]: My hope is to clarify the target audience of Community Edition relative to our current Hubs Cloud userbase which is used to having the AWS console and interfaces guiding them through the deployment process. CE requires a lot more familiarity with all hubs services, command line tools, and potentially multiple cloud platforms. I do agree that we could find a better phrasing to 'complex application infrastructure' to make it clear that I am referring to cloud platforms.

File: community-edition/readme.md (hunk 1 above)
Comment (mikemorran) [reply community-edition/readme.md]: Would it be better to say "assisting developers as they build, deploy, and scale..."?

File: community-edition/readme.md (hunk 7 above)
Comment (mikemorran) [reply community-edition/readme.md:32]: I think my problem with the current formatting is that the commands are in bulleted list. The example you shared has specific formatting for code snippets which we don't currently have here

**Review by mikemorran — COMMENTED** [review]
LGTM for now, but lets follow up later

**Pull request merged**